src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -63,8 +63,8 @@
 
   public ConsumeService(Map<String, Object> props, String name) throws Exception {
     _name = name;
-    Map consumerPropsOverride = props.containsKey(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG) ?
-        (Map) props.get(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG) : new HashMap<>();
+    Map consumerPropsOverride = props.containsKey(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG)
+      ? (Map) props.get(ConsumeServiceConfig.CONSUMER_PROPS_CONFIG) : new HashMap<>();
```

Comment:
I was following LinkedIn style guidelines but while it is a LinkedIn project, it is opensource so perhaps it can diverge. What do you think?